MIP/mip-21/README.md

```diff
@@ -0,0 +1,91 @@
+
+We believe the best approach is to draw from live implementations that have achieved a streamlined user experience with minimal cost to the user. One such example is USDC's [CCTP](https://www.circle.com/en/cross-chain-transfer-protocol), which offers simplicity, security, and efficiency in cross-chain bridging. Instead of increasing the complexity of our current bridge design, we should aim to adopt this proven mechanism.
+
+## Specification
+
+The Cross-Chain Transfer Protocol (CCTP) operates using a set of attestors that validate all bridge transactions occurring between chains. These attestors listen for bridge events, confirm the validity of transactions, and, if necessary, revert them by providing appropriate proofs. A transfer allowance managed by a centralized authority further ensures that breaches are mitigated by limiting the amount of value transferred.
+
+To align with this approach and reduce complexity:
+
+- **Bridge Relayer as Attestor**: The Bridge Relayer in our system will function as an attestor, operating via a multisig scheme (i.e. the attestor collects signatures from a threshold number of comittee members that approve of the message). This relayer, through aggregated signatures, produces a proof on a signed message.  
+
+- **User Interaction**: Users, or other parties interested in completing the bridge, retrieve their funds on the target chain by submitting the signature and message.
+
+- **Validation Mechanism**: The off-chain signature produced by the multisig scheme is verified against the message on-chain.
+The challenge is to make sure that the signature is only used once, but the beauty is that the signature is provided by a multi-party system that do not need to provide onchain signatures.
+Relying on a protocol like CCTP that has been tested in live environments significantly reduces the risk of any unexpected utilization of off-chain and on-chain proofs.  
+
+Or on Circle's own words:
+
+1. An on-chain component on source domain emits a message.
+2. Circle's off-chain attestation service signs the message.
+3. The on-chain component at the destination domain receives the message, and forwards the message body to the specified recipient.
+
+This approach minimizes the need for additional components and avoids the reliance on synthetic assets, leading to a simplified and cost-effective bridge solution.
+
+### Flow
+
+1. **User Deposits**:  
+   - The user deposits tokens on the source chain, which triggers a deposit event.
+   
+2. **Bridge Relayer**:  
+   - The Bridge Relayer, listening for the deposit event, picks up the message or messages associated with the deposit.
+   
+3. **Multisig Signing**:  
```

Comment:
Are you thinking of using Iris for this? I think in general it's fine to draw inspiration from CCTP but more specifics about implementation on Movement would be helpful to get a sense of what is entailed.

Reply:
No, I think Iris is also closed source.

Reply:
Do you have some other attestation mechanism in mind?

Reply:
closing this, as 
- the attester may not be the same as FFS.
- this is propossed now as an MD and thus does not require further specification